pool_configs/examples/kmd.json

```diff
@@ -28,6 +28,9 @@
         "ca": ""
     },
 
+    "explorer": "",
+    "_comment_explorer": "Explorer domain without https:// or ending /",
```

Comment:
Unfortunately, many coins come out with http-only explorers. We need to be able to define that in the config.